src/applications/check-in/pre-check-in/pages/Confirmation/index.jsx

```diff
@@ -16,53 +16,57 @@ const Confirmation = () => {
   const { appointments } = useSelector(selectVeteranData);
   const { data: formData } = useSelector(selectFormData);
   const hasUpdates = Object.values(formData).includes('no');
-  return (
-    <div
-      className="vads-l-grid-container vads-u-padding-bottom--3 vads-u-padding-top--3"
-      data-testid="confirmation-wrapper"
-    >
-      <h1 tabIndex="-1" className="vads-u-margin-top--2">
-        You’ve completed pre-check-in
-      </h1>
-      <AppointmentBlock appointments={appointments} />
-      {hasUpdates ? (
-        <va-alert
-          background-only
-          status="info"
-          show-icon
-          data-testid="confirmation-update-alert"
-        >
-          {/** TODO INFO ICON */}
```

Comment:
Not related to this ticket, but the inline comment isn't needed anymore. This is one I should have removed when I updated the the icon and title last week.